theano/tensor/nnet/tests/test_conv.py

```diff
@@ -281,8 +281,6 @@ def test_unroll_batch_kern_fail(self):
     @attr('slow')
     def test_subsample(self):
         # Tests convolution where subsampling != (1,1)
-
-        self.validate((3, 2, 7, 5), (5, 2, 2, 3), 'valid', subsample=(2, 2))
```

Comment:
That is not a duplicate test and should be kept.

Reply:
Yes, I had some conflicts and I thought, that my version was correct, so I merged my version. I figured it later out when the tests failed on Jenkins and saw where the problem was.

Now everything is correct, right? Anything else to do?

Reply:
@abergeron Is there anything left to do? The change you requested has been dealt with by the master rebasing...